pkg/ebpf/tracer_windows.go

```diff
@@ -177,20 +146,17 @@ func (t *Tracer) GetStats() (map[string]interface{}, error) {
 	}
 
 	stateStats := t.state.GetStats()
-
-	return map[string]interface{}{
-		"state":                    stateStats,
-		"total_flows":              driverStats["total_flows"],
-		"open_flows":               driverStats["open_flows"],
-		"closed_flows":             driverStats["closed_flows"],
-		"more_data_errors":         driverStats["more_data_errors"],
-		"driver_total_flow_stats":  driverStats["driver_total_flow_stats"],
-		"driver_flow_handle_stats": driverStats["driver_flow_handle_stats"],
-	}, nil
+	stats := map[string]interface{}{
+		"state": stateStats,
+	}
+	for _, name := range network.DriverExpvarNames {
+		stats[string(name)] = driverStats[name]
+	}
+	return stats, nil
```

Comment:
IMO this is harder to follow than hardcoding the names.

Reply:
The names were hardcoded in multiple places before, and thus was possible to have them get out-of-sync (it happened to me). This ensures there is never a mismatch.